core/dbt/contracts/graph/unparsed.py

```diff
@@ -242,6 +242,7 @@ class Quoting(dbtClassMixin, Mergeable):
 
 @dataclass
 class UnparsedSourceTableDefinition(HasColumnTests, HasTests):
+    config: Dict[str, Any] = field(default_factory=dict)
```

Comment:
There is a HasConfig mixin class which some of the other Unparsed nodes use. If it's possible to use that (sometimes you run into precedence issues with fields without defaults) it probably should for consistency.

Reply:
We ran into issues with defaults when we tried to use it.

Reply:
I'm not surprised. Getting the default and non-default configs lined up is a headache.